src/main/puzzle/tutorial/skill-tally-item.gd

```diff
@@ -62,14 +63,10 @@ func increment() -> void:
 	_update_label()
 
 
-"""
-Connects the signals in 'signal_names' to the appropriate puzzle nodes.
-"""
-func _connect_signals() -> void:
-	if Engine.is_editor_hint():
-		return
```

Comment:
Preserve this comment; `Connects the signals in 'signal_names' to the appropriate puzzle nodes.`

Reply:
Preserved comment; `Connects the signals in 'signal_names' to the appropriate nodes.`